c10/core/DispatchKeySet.h

```diff
@@ -203,6 +207,11 @@ constexpr DispatchKeySet autograd_dispatch_keyset = DispatchKeySet({
     DispatchKey::AutogradOther,
 });
 
+// See Note [TLS Initialization]
+constexpr DispatchKeySet InplaceOrView_keyset = DispatchKeySet({
```

Comment:
What if we called this `default_included_keyset` and described our scheme in more general terms? Like, "To preserve the invariant that raw TLS storage of the default state is zero, we obtain the *actual* current include keyset by xoring raw storage with the default state defined here". It's actually not a bad setup, and framing it this way would clarify how to add more nonzero default bits in the future if necessary (as well as using the same recipe if we needed other nonzero initial TLS values).